src/files/modals/bulk-import-modal/BulkImportModal.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Button from '../../../components/button/button.tsx'
+import { Modal, ModalActions, ModalBody } from '../../../components/modal/Modal.js'
+import { withTranslation } from 'react-i18next'
+import * as isIPFS from 'is-ipfs'
+import Icon from '../../../icons/StrokeDocument.js'
+import { normalizeFiles } from '../../../lib/files.js'
+
+class BulkImportModal extends React.Component {
+  static propTypes = {
+    onCancel: PropTypes.func.isRequired,
+    onBulkCidImport: PropTypes.func.isRequired,
+    className: PropTypes.string
+  }
```

Comment:
we should remove prop-types. Eventually, we're moving this whole repo to typescript and will get type safety during transpilation.